### ISSUE TITLE
[WHL][VxWorks] Launch VxWorks fails due to no suitable video mode found error

### ISSUE THREAD

**Issue by fuzhongl** [projectacrn/acrn-hypervisor#5151]
**Platform**
WHL

**Codebase**
v2.1

**Scenario**
 Industry

Steps:
1. Launch SOS
2. Use the script to launch Vxworks.

Expected Result:
Launch vxworks success.

Actual result:
Launch vxworks fails due to error detected from the vxworks grub:

**Comment by fuzhongl** [comment]
[External_System_ID] ACRN-6557

**Comment by gvancuts** [comment]
Error thrown when attempting to lauch VxWorks:
```
WARNING: no console will be available to OS
error: no suitable video mode found.
initialize CPU
start CPU
initialize MMU
initialize base virtual memory support
initialize address space library
initialize global mapping
initialize VxBus
        install bus types:
                vxbHpetBus(Intel Architecture HPET Bus)
                vxbMiiBus(MII bus type)
```

The problem was introduced with this PR #4692

**Comment by fuzhongl** [comment]
fixed with v2.2
